src/commands/moderations.js

```diff
@@ -44,40 +45,81 @@ command.execute = async (message, args, database, bot) => {
     return;
   }
 
-  let text = '', i = 1;
-  async function send(start, end) {
-    let embed = new Discord.MessageEmbed({
-      author: {
-        name: `Moderations for ${user.username}#${user.discriminator} (${start} to ${end} of ${moderations.length})`,
-        iconURL: user.avatarURL()
-      },
-      description: text
-    });
-    await message.channel.send(embed);
-  }
-  for (let [key,/** @type {ModerationData} */ moderation] of moderations.entries()) {
-    if (text.length > 1800) {
-      await send(i, key);
-      text = '';
-      i = key + 1;
+  let index = 0, lastModified = Date.now();
+  /** @type {module:"discord.js".Message} */
+  const response = await message.channel.send(generateEmbed(moderations,user,0));
+
+  await response.react(util.icons.left)
+  await response.react(util.icons.right)
+
+  const reactionCollector = response.createReactionCollector(async (reaction, user) => {
+    const member = message.guild.member(user);
+    return await util.isMod(member) || member.hasPermission('MANAGE_MESSAGES') && [util.icons.left,util.icons.right].includes(reaction.emoji.name)
```

Comment:
I think only the user that sent the original message should be able to react here. Also, any other reaction should be removed, not just ignored.